aws/lambda/benchmark_regression_summary_report/common/config_model.py

```diff
@@ -223,6 +224,30 @@ def get_github_notification_config(self) -> Optional[GitHubNotificationConfig]:
         return GitHubNotificationConfig.from_dict(self.notification_config)
 
 
+@dataclass
+class ReportConfig:
+    """
+    decide what to include in db summary report
+    report_level: lowest level of regression to store in db
+    """
+
+    report_level: Literal[
+        "no_regression", "suspicious", "regression", "insufficient_data", "none"
+    ] = "regression"
+
+    def get_report_orders(self) -> dict[str, int]:
+        return {
+            "no_regression": 0,
```

Comment:
The list here is rather confusing to look at, maybe some comments will help

1. `no_regression` and `none` are at the two end of the spectrum.  What would `none` be in this case? I guess you mean no data
2. It's easier to map these reports by the order of severity, so maybe `no_regression` → `insufficient_data` → `suspicious` → `regression`